ROADMAP.md

```diff
@@ -1,109 +0,0 @@
-# Apollo Server Roadmap
```

Comment:
It might make sense to keep this file around but have it say something akin to "This page most recently described the roadmap for Apollo Server 3. With the release of Apollo Server 3, we are now in the process of updating it to describe future work"?

Reply:
Sure, though I don't know if it will end up in this file. But seems reasonable.